lib/api/tenure/v1/service.ts

```diff
@@ -83,13 +83,14 @@ export const editTenure = async ({ id, ...data }: EditTenureParams): Promise<voi
   return response.data;
 };
 
-export const editTenuredAsset = async ({
-  id,
-  ...data
-}: EditTenuredAssetParams): Promise<void> => {
-  const response = await axiosInstance.patch(
-    `${config.tenureApiUrlV1}/tenures/${id}`,
-    data,
-  );
-  return response.data;
+export const editTenuredAsset = async (
+  id: string,
+  tenuredAssetUpdate: EditTenuredAssetParams,
```

Comment:
The request name is editTenuredAsset but the endpoint it'd hit with that URL would be EditTenureDetails in Tenure API. I see editTenure sends a request to the same exact endpoint, with the only difference being that the body/payload uses a different interface. Is this what we want?

To add to that, looking at the backend object I see the object we're sending to Tenure API > Controller > EditTenureDetails is not the same as the object we're expecting in the endpoint (EditTenureDetailsRequestObject).

Will the controller also be changed?

![image](https://github.com/LBHackney-IT/mtfh-frontend-common/assets/70756861/c30fa86a-17ff-4abe-9d39-3a98859b7d48)

Reply:
So in a nutshell are you suggesting - add tenuredAsset to TenureParams and use the existing editTenure call?

Reply:
> So in a nutshell are you suggesting - add tenuredAsset to TenureParams and use the existing editTenure call?

I wasn't suggesting that but to be honest as you mention it it doesn't seem a bad idea at all. I'd consider adding `tenuredAsset?` to `EditTenureParams` and then yes, keeping just editTenure. This way you could remove the `EditTenuredAssetParams` interface and the API call that uses it (`editTenuredAsset`).